kidviz/views.py

```diff
@@ -156,45 +174,32 @@ def get(self, request, *args, **kwargs):
         return super().get(request, *args, **kwargs)
 
     def post(self, request, *args, **kwargs):
```

Comment:
I add this comment here because there is no appropriate place for it.

Label is setup should inform that this is not normal setup, but reconfiguration.